src/background/utils.js

```diff
@@ -1,13 +1,34 @@
+import { IS_FIREFOX } from '../lib/environment';
 import {
-  setContractInstance,
-  contractCall,
+  CONNECTION_TYPES,
   getAddressByNameEntry,
   watchUntilTruthy,
 } from '../popup/utils';
 import store from './store';
 
+const BACKEND_AE_VM = 'aevm';
+const BACKEND_FATE = 'fate';
+
 let tippingContract;
 
+export const setContractInstance = async (tx, sdk, contractAddress = null) => {
+  let contractInstance = false;
```

Comment:
This method was moved 1:1 from file `helper.js` (line 94). To be honest I would like to avoid messing around with this code within this PR. Maybe you can create a task for suggested changes?